hack/release/release.sh

```diff
@@ -1,20 +1,20 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-GIT_TAG=$(git describe --exact-match --tags || echo "no tag")
-if [[ "$GIT_TAG" == "no tag" ]]; then
-    echo "Failed to release: commit is untagged"
-    exit 1
-fi
-HEAD_HASH=$(git rev-parse HEAD)
-
-SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
+SCRIPT_DIR="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
+# shellcheck source=hack/release/common.sh
 source "${SCRIPT_DIR}/common.sh"
-config
+
+git_tag="$(git describe --exact-match --tags || echo "no tag")"
+if [[ "${git_tag}" == "no tag" ]]; then
```

Comment:
What's the reason this conditional check is different from the one that we use for `release-crd.sh`?

Reply:
This is guard clause and the other is a conditional extra function. I didn't modify these checks, just refactored the bash style.